packages/action-button/src/action-button.css

```diff
@@ -44,6 +44,25 @@ governing permissions and limitations under the License.
     text-align: var(--spectrum-actionbutton-label-text-align);
 }
 
+:host([size='xs']) {
+    /* Work around non-square icon only Action Buttons in Spectrum CSS */
+    min-width: var(--spectrum-actionbutton-height, 0);
+
+    /* Pass t-shirt sizing to icons within the XS-sized Action Button */
+    --spectrum-icon-tshirt-size-height: var(
+        --spectrum-alias-workflow-icon-size-xs
+    );
+    --spectrum-icon-tshirt-size-width: var(
+        --spectrum-alias-workflow-icon-size-xs
+    );
+    --spectrum-ui-icon-tshirt-size-height: var(
+        --spectrum-alias-ui-icon-cornertriangle-size-75
+    );
+    --spectrum-ui-icon-tshirt-size-width: var(
+        --spectrum-alias-ui-icon-cornertriangle-size-75
+    );
+}
+
```

Comment:
A Q for my own understanding: I would have expected to see CSS for the XS size in the upstream `spectrum-action-button.css` rather than in a local override. Is XS a special case or am I misunderstanding what comes from spectrum-foo.css vs foo.css?